### ISSUE TITLE
Cannot find EcsComponent in namespace Invert.uFrame.ECS

### ISSUE THREAD

**Issue by grofit** [micahosborne/uFrame#2]
Seem to get an error when trying to build on an empty project with a single component, here were the steps:

- Create a new project
- Downloaded and unzipped repo into assets
- Create a new module
- Add component called `Buffable` (could be anything, it had no content in it)
- Pressed the build button

Then got the error:

```
Assets/Plugins/Editor/uFrame.ECS.Editor/Components/Buffable.cs(24,48): error CS0234: The type or namespace name `EcsComponent' does not exist in the namespace `Invert.uFrame.ECS'. Are you missing an assembly reference?
```

I am using unity 5.3.4f and like I say it is a new project with nothing but this repo.

**Comment by mkuhr** [comment]
Only on Github: Not for the Unity Asset. We have the uframeECS Architect and Editor sources so on default those are open:
<h3> Thats how you use it </h4>
<ul>
<li>Step 1: You have to create a new database first, when you pull from Github. For that click "Database" in the lower left corner and choose "Manage" </li>
<li> Step 2: Click on "New Empty Database". Choose a name and leave Codepath as is</li>
<li>Step 3: Create Workspace with your name</li>
<li>Step 4 : Create Module with your Project Name</li>
<li> Step 5: Have fun. Create components, systems, events..... </li>
</ul>

**Comment by grofit** [comment]
So based upon this I would probably say it should default to not being in a DB so you start at the "Make a New Database" screen, as unless you know the tools and have read this message you will probably end up doing what I did and think "it doesn't work"

**Comment by mkuhr** [comment]
Easy and fast solution proposition: 
Change  
`var dbDirectories = Directory.GetDirectories(path,"*.db",SearchOption.AllDirectories);`
 to 
`var dbDirectories = Directory.GetDirectories(path,"*.db",SearchOption.TopDirectoryOnly);`

Easy start for using ECS. With the normal Flow:

1. Create Database

Pro Users can undo this to make the ECS Database visible in the uFrame Grapheditor to change the Implementation?

**Comment by grofit** [comment]
I dont know the knock-on effects this would cause but as long as it makes workflow for getting started with ECS easier and error free then I would go for it.

:+1:

**Comment by micahosborne** [comment]
This would only allow databases to be created/used in the root directory.  You really just need to create another database and ignore the ECS database.  I get that might be confusing, but hey didn't you learn from it? :p

**Comment by micahosborne** [comment]
This would only allow databases to be created/used in the root directory.  You really just need to create another database and ignore the ECS database.  I get that might be confusing, but hey didn't you learn from it? :p

**Comment by micahosborne** [comment]
This would only allow databases to be created/used in the root directory.  You really just need to create another database and ignore the ECS database.  I get that might be confusing, but hey didn't you learn from it? :p

**Comment by micahosborne** [comment]
This would only allow databases to be created/used in the root directory.  You really just need to create another database and ignore the ECS database.  I get that might be confusing, but hey didn't you learn from it? :p

**Comment by grofit** [comment]
You are welcome to class it as closed and just put it down as a documentation step, or just ignore it completely, I am just aware that *most* people looking to try it will probably fall over at the first point without knowing this first.

**Comment by mkuhr** [comment]
I agree with grofit. Thats not user friendly enough. Too many people will just abandon it at this stage, when they are trying it out. And this first experience doesnt live up to this fantastic product.
 
Had to edit my "making it work" above, because you even have to open a workspace before manage database window shows up when you click it.

Whats wrong with having databases at the root directory for starters?

Or can you point me to a solution where i can have all Directories searched, but show the "Create New Database Window" without having to open any workspace.
